.nvmrc

```diff
@@ -0,0 +1 @@
+22
```

Comment:
I don't use nvm. The nvmrc file is understood by a few other tools, but I don't think create-single-spa should assume using nvm